webapp/src/clj/lipas/backend/core.clj

```diff
@@ -804,7 +829,20 @@
 
 (defn enrich-loi
   [{:keys [geometries] :as loi}]
-  (assoc-in loi [:search-meta :geometries] (feature-coll->geom-coll geometries)))
+  (let [geom-coll (feature-coll->geom-coll geometries)]
+    (-> loi
+        (assoc-in [:search-meta :location :geometries] geom-coll)
+        (assoc-in [:search-meta :location :wgs84-point] (-> (gis/->flat-coords geometries)
+                                                            first)))))
+
+(defn search-lois-with-params
```

Comment:
I think we should remove `:search-meta` key from result docs. You can either exclude it in the ES-query or dissoc it from the `:_source` in result docs. The first one might be more efficient.  Example how to add "source exclusions" can be found here: https://github.com/lipas-liikuntapaikat/lipas/blob/master/webapp/src/cljs/lipas/ui/search/events.cljs#L489